### DISCUSSION TITLE
How to play multiplayer?

### DISCUSSION THREAD

**Discussion by printhelloworldasdas** [nzp-team/nzportable#921] [General]
Hello, im triying to play multiplayer but i still in the joining screen is can be a problem of the internet?

**Comment by MotoLegacy** [comment]
yes

**Reply by printhelloworldasdas** [reply]
ok thx i think is the wifi of my school lol

**Reply by printhelloworldasdas** [reply]
all the wifis of all the schools i was is a trash xd

**Reply by printhelloworldasdas** [reply]
a and the game is blocked for me i had to fork the game
